src/components/Onboard.tsx

```diff
@@ -62,54 +63,58 @@ function SignInForm() {
  */
 function Onboard(props: RouteComponentProps) {
   const currentUser = useSelector((state: RootState) => state.auth.userUID);
-  const history = useHistory();
-
+  const [redirectDestination, setRedirect] = useState("/");
```

Comment:
Using a state variable just adds another layer of complexity. If we need to redirect, then we can just return the redirect element immediately.